tools/pipelines/templates/download-pipeline-artifact.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# download-pipeline-artifact pipeline
+
+parameters:
+- name: pipelineDefinition
+  type: string
+
+- name: branchName
+  type: string
+  
+steps:
+- checkout: none
+  clean: true
```

Comment:
This step doesn't need to be in this template, right? It's done once at the beginning of the upload-json-steps file, and afterwards each inclusion of this file just needs to download the pipeline artifact.